trailmap/pathway_analysis.py

```diff
@@ -0,0 +1,116 @@
+import networkx as nx
+import matplotlib.pyplot as plt
+from pprint import pprint
+import pygraphviz as pgv
+
+
+def draw_graph(G):
+    pos = nx.drawing.nx_pydot.graphviz_layout(G, prog=dot)
+
+    plt.figure()
+    # nx.draw_shell(G, with_labels=True)
+    nx.draw_networkx_nodes(G, pos)
+    nx.draw_networkx_labels(G, pos)
+    nx.draw_networkx_edges(G, pos)
+    # nx.draw_networkx_edge_labels(G, pos)
+
+    return plt
+
+
+def draw_path(G, path):
+    draw_graph(G)
+    pos = nx.drawing.nx_pydot.graphviz_layout(G)
+    nx.draw_networkx_nodes(G, pos=pos, nodelist=path, node_color="red")
+    plt.show()
+    return
+
+
+def print_graph_parameters(G):
```

Comment:
might be interesting to brainstorm on parameters of interest:
* depth of graph (i.e. longest path)
* shortest path
* number of pathways

Reply:
Yeah those are so simple that it doesn't make sense to make people specifically request them, you're right I should just print those as part of the general graph parameters. Even that I currently have set as a command line flag, do you think it's important enough that I should just remove that flag and auto-print it?